packages/react/src/components/List/HierarchyList/HierarchyList.jsx

```diff
@@ -87,6 +90,7 @@ const defaultProps = {
   hasSearch: false,
   hasPagination: true,
   hasMultiSelect: false,
+  hasDeselection: false,
```

Comment:
We are changing the default  behavior for this component which could lead to breaks for consumers. Can we rework this so the old behavior is maintained? `hasNoDeselection` seems gross but could be an alternative. Otherwise we need to default this to true and add to our V3 issues checklist to change this to false in the new version.

Reply:
Yeah I'm not a fan of `hasDeselection` as a name either honestly. This was a mistake, meant it to be defaulted to true